packages/manager/apps/dedicated/client/app/account/user/users/users.controller.js

```diff
@@ -75,4 +86,32 @@ export default class UserAccountUsersCtrl {
   onTransformItemDone() {
     this.usersLoading = false;
   }
+
+  onTransformGroup(groupName) {
+    this.groupsLoading = true;
+    return this.groupsService.getGroup(groupName).then((group) => {
+      const shortDescription =
+        group.description != null &&
+        group.description.length > this.descriptionMaxSize
+          ? `${group.description.substring(0, this.descriptionMaxSize)}...`
+          : group.description;
+      set(group, 'shortDescription', shortDescription);
+      return group;
+    });
+  }
+
+  onTransformGroupDone() {
+    this.groupsLoading = false;
+  }
+
+  initIdentityProvider() {
+    this.usersService
+      .getIdentityProvider()
+      .then((identityProvider) => {
+        this.identityProvider = identityProvider;
+      })
+      .catch(() => {
+        this.identityProvider = null;
+      });
+  }
```

Comment:
I duplicated it because they're used in different contexts so this isn't necessary the same code.
For example, the behaviour of initIdentityProvider() of the sso-update-controller is slightly different now.